cluster-autoscaler/core/scaledown/actuation/actuator_test.go

```diff
@@ -545,17 +545,17 @@ func TestStartDeletion(t *testing.T) {
 					PodEvictionResults: map[string]status.PodEvictionResult{
 						"drain-node-0-pod-0": {Pod: removablePod("drain-node-0-pod-0", "drain-node-0"), Err: cmpopts.AnyError, TimedOut: true},
 						"drain-node-0-pod-1": {Pod: removablePod("drain-node-0-pod-1", "drain-node-0"), Err: cmpopts.AnyError, TimedOut: true},
-						"drain-node-0-pod-2": {Pod: removablePod("drain-node-0-pod-2", "drain-node-0")},
+						"drain-node-0-pod-2": {Pod: removablePod("drain-node-0-pod-2", "drain-node-0"), GracePeriodSeconds: apiv1.DefaultTerminationGracePeriodSeconds},
 					},
 				},
 				"drain-node-1": {ResultType: status.NodeDeleteOk},
 				"drain-node-2": {
 					ResultType: status.NodeDeleteErrorFailedToEvictPods,
 					Err:        cmpopts.AnyError,
 					PodEvictionResults: map[string]status.PodEvictionResult{
-						"drain-node-2-pod-0": {Pod: removablePod("drain-node-2-pod-0", "drain-node-2")},
+						"drain-node-2-pod-0": {Pod: removablePod("drain-node-2-pod-0", "drain-node-2"), GracePeriodSeconds: apiv1.DefaultTerminationGracePeriodSeconds},
 						"drain-node-2-pod-1": {Pod: removablePod("drain-node-2-pod-1", "drain-node-2"), Err: cmpopts.AnyError, TimedOut: true},
-						"drain-node-2-pod-2": {Pod: removablePod("drain-node-2-pod-2", "drain-node-2")},
+						"drain-node-2-pod-2": {Pod: removablePod("drain-node-2-pod-2", "drain-node-2"), GracePeriodSeconds: apiv1.DefaultTerminationGracePeriodSeconds},
```

Comment:
Could you add some test cases where this value is something different than the default?

Reply:
Added a test case to `drain_test.go` (since the original changes were in `drain.go`), and added a test to `actuator_test.go` that exercises the deletion in the context of `StartDeletion`.  Let me know if these are ok / if additional test cases are needed!